listenbrainz/webserver/static/js/src/utils/types.d.ts

```diff
@@ -516,6 +516,12 @@ declare type UserTrackRecommendationMetadata = {
   recording_msid: string;
 };
 
+/** For recommending a track personally from the front-end */
+declare type UserTrackPersonalRecommendationMetadata = UserTrackRecommendationMetadata & {
+  blurb_content: string | null;
+  users: Array<string> | null;
```

Comment:
These should probably be:
I assume blurb content could be an empty string rather than null, and same for an empty users array
```suggestion
  blurb_content: string;
  users: Array<string>;
```